ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -78,3 +78,15 @@ def normalize(tensor, ord="euclidean", axis=None, name=None):
     "float32",
     "float64",
 )
+
+
+def l2_normalize(x, axis=None, epsilon=1e-12, name=None):
+    square_sum = ivy.sum(ivy.square(x), axis=axis, keepdims=True)
+    x_inv_norm = ivy.reciprocal(ivy.sqrt(ivy.maximum(square_sum, epsilon)))
+    return ivy.multiply(x, x_inv_norm)
+
+
+l2_normalize.supported_dtypes = (
```

Comment:
We now use decorators to assign `unsupported , supported dtypes and devices` . You are supposed to use `with_supported_dtypes` for this. Check this link https://lets-unify.ai/ivy/deep_dive/data_types.html#backend-data-type-bugs on how to add the version